### ISSUE TITLE
UnusedFormalParameter false positive

### ISSUE THREAD

**Issue by EvgenyOrekhov** [phpmd/phpmd#561]
```php
<?php

class Example
{
    public function func($param)
    {
        return "${param}";
    }
}
```

> /app/src/test.php:5	Avoid unused parameters such as '$param'.

**Comment by fgm** [comment]
Same problem with a local variable:

```php
<?php
function foo() {
  $bar = 'baz';
  return "${bar}"
}
```

> a.php:3   Avoid unused local variables such as '$bar'.

**Comment by ravage84** [comment]
Closed through merging #575.
Thank you @EvgenyOrekhov & @fgm for reporting.
